storage/rocksdb/ha_rocksdb.cc

```diff
@@ -1682,6 +1690,9 @@ class Rdb_transaction {
   */
   virtual bool is_writebatch_trx() const = 0;
 
+  virtual rocksdb::Status Merge(rocksdb::ColumnFamilyHandle *cf,
```

Comment:
let's stick with the naming convention of MyRocks, rather than RocksDB, hence use `merge()` as the method name, since this class is not derived from any RocksDB class